CHANGELOG.md

```diff
@@ -1,3 +1,255 @@
+## v0.6.0 - 2021-01-05
+
+We're happy to announce the release of Lokomotive v0.6.0 (Flying Scotsman).
+
+This release includes several new features, many component updates, and a new platform - [Tinkerbell](https://tinkerbell.org/).
+
+### Changes in v0.6.0
+
+#### Kubernetes updates
+
+- Update Kubernetes to v1.19.4 and AKS to v1.18.10 ([#1189](https://github.com/kinvolk/lokomotive/pull/1189)).
+
+#### Component updates
+
+- Update `external-dns` to v0.7.4 ([#1115](https://github.com/kinvolk/lokomotive/pull/1115)).
+- Update `metrics-server` to v2.11.2 ([#1116](https://github.com/kinvolk/lokomotive/pull/1116)).
+- Update `cluster-autoscaler` to version v1.1.0 ([#1137](https://github.com/kinvolk/lokomotive/pull/1137)).
+- Update `rook` to v1.4.6 ([#1117](https://github.com/kinvolk/lokomotive/pull/1117)).
+- Update `velero` to v1.5.2 ([#1131](https://github.com/kinvolk/lokomotive/pull/1131)).
+- Update `openebs-operator` to v2.2.0 ([#1095](https://github.com/kinvolk/lokomotive/pull/1095)).
+- Update `contour` to v1.10.0 ([#1170](https://github.com/kinvolk/lokomotive/pull/1170)).
+- Update `experimental-linkerd` to stable-2.9.0 ([#1123](https://github.com/kinvolk/lokomotive/pull/1123)).
+- Update `web-ui` to v0.1.3 ([#1237](https://github.com/kinvolk/lokomotive/pull/1237)).
+- Update `prometheus-operator` to v0.43.2 ([#1162](https://github.com/kinvolk/lokomotive/pull/1162)).
+- Update Calico to v3.17.0 ([#1251](https://github.com/kinvolk/lokomotive/pull/1251)).
+- Update `aws-ebs-csi-driver` to v0.7.0 ([#1135](https://github.com/kinvolk/lokomotive/pull/1135)).
+
+#### Terraform provider updates
+
+- Update Terraform providers to their latest versions ([#1133](https://github.com/kinvolk/lokomotive/pull/1133)).
+
+#### New platforms
+
+- Add support for Tinkerbell platform ([#392](https://github.com/kinvolk/lokomotive/pull/392)).
+
+#### Bug fixes
+
+- Add new worker pools when TLS bootstrap is enabled without remaining stuck in the installation phase ([#1181](https://github.com/kinvolk/lokomotive/pull/1181)).
+- `contour`: Consistently apply node affinity and tolerations to all scheduled workloads ([#1161](https://github.com/kinvolk/lokomotive/pull/1161)).
+- Equinix Metal: Ensure Calico CRDs get created before use ([#1177](https://github.com/kinvolk/lokomotive/pull/1177)). They were occasionally failing.
```

Comment:
This doesn't need to be there. v0.5.0 does not include `--wait` yet.

Reply:
What do you mean v0.5.0 does not include wait? Do you mean it is implemented partially and the user may not see the full benefits just yet?

Reply:
No. I mean users using v0.5.0 are not affected by occasionally failing Calico CRs, because --wait was added after v0.5.0 release.

If buggy --wait would be added in v0.5.0, then this fix should be included in the changelog.

Reply:
You mean that --wait was added 0.5.0 and this issue with calico failing was fixed immediately. So we don't need to mention it? I don't know if regressions fixed before release should be part of the release notes or not!

Reply:
>You mean that --wait was added 0.5.0 and this issue with calico failing was fixed immediately. So we don't need to mention it? 

Yes, Issue with --wait is not part of v0.5.0 tag/version, so I suggest we remove this from release notes.

>I don't know if regressions fixed before release should be part of the release notes or not!

IMO it shouldn't. This is why you have git history and releases. To reduce amount of history (release notes) for the user to go trough and to have a chance to fix bugs before users notice :)